spec/jobs/emit_web_hook_event_spec.rb

```diff
@@ -239,7 +239,7 @@
       stub_request(:post, post_hook.payload_url)
         .to_return(body: 'OK', status: 200)
 
-      WebHookEventType.all.pluck(:name).each do |name|
+      WebHookEventType.limit(10).pluck(:name).each do |name|
```

Comment:
Can you clarify what this is doing? Only emitting core webhooks?

Reply:
Correct. It is only selecting the first 10 webhook event types which all happen to be in core.

Reply:
For this to be a "good" test I don't think it even needs to loop through ALL of them. It could probably just grab the first one and validate that this functionality is working.

Reply:
I think I would prefer that rather than having 10 as a constant here.